### PR TITLE
Integrated the Login module.

### PR DESCRIPTION
**Pull request by RajuKoushik**

### CODE REVIEW

File: scanapp/models.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 from __future__ import unicode_literals
 
 from django.conf import settings
+from django.contrib.postgres.fields import JSONField
```
Comment (singh1114) [inline scanapp/models.py:27]: No need to add them again. If you want to, delete your commit.

Comment (RajuKoushik) [reply scanapp/models.py:27]: I'm having a problem with squashing the 2 commits

File: scanapp/templates/scanapp/login.html [hunk 2]
```diff
@@ -0,0 +1,278 @@
+{% load staticfiles %}
+<html>
```
Comment (singh1114) [inline scanapp/templates/scanapp/login.html:3]: Extend scanapp/base.html

Please do not add HTML and head tags.

Comment (RajuKoushik) [reply scanapp/templates/scanapp/login.html:3]: Okay

**Pull request merged**